src/constants/categories.js

```diff
@@ -252,7 +252,7 @@ export const CATEGORIES = [
     ],
     subCategories: [
       {
-        name: 'Employment',
+        name: 'Labour Force',
```

Comment:
We should be sentence case everywhere, so "Labour force".

Actually "Safety and Crisis Support" under Liveability should be fixed too.

Maybe ask Nick about the category titles "Liveability and Sustainability", "Jobs & Skills", and "Infrastructure and Investment". I would think these should all be sentence case too.

Reply:
I am all for sentence case everywhere. There seems to be a lot of random capitalisations in titles, so thought that was perhaps the vibe we were going for.

Reply:
What about "All Cities" - another case for sentence case I reckon?

@davidgilbertson

Reply:
Have updated that and all the others I could find.